test/build/55_cache.bats

```diff
@@ -1399,3 +1399,34 @@ EOF
     [[ $output = *'deleting, see issue #1351: var/lib/rpm/__db.001'* ]]
     [[ ! -e $CH_IMAGE_STORAGE/img/tmpimg/var/lib/rpm/__db.001 ]]
 }
+
+@test "${tag}: restore ACLs, xattrs" {  # issue #1287
+    ch-image build-cache --reset
+    ch-image build -t tmpimg - <<'EOF'
+FROM alpine:3.17
+RUN apk add attr
+RUN apk add acl
+RUN touch /home/foo
+RUN setfattr -n user.foo -v bar /home/foo
+RUN setfacl -m u:root:r /home/foo
```

Comment:
Why do we build the same image twice? Is it really supposed to be the same? Explain why, and only write out the Dockerfile once (can put it in a variable I think).